swyft/store/store.py

```diff
@@ -215,7 +225,7 @@ def __getitem__(self, i):
         par = self.v[i]
         return (sim, par)
```

Comment:
as a note to myself, I want to clarify which kind of parameter we return by explicitly calling it v